tests/p2p_multiple_listen_test.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
```

Comment:
thanks for adding the test.
It is a bit confusing, can you add more comments about its actions?

also, now `net_plugin`'s options `p2p-listen-endpoint` and `p2p-server-address` could be a vector. So I think we should have a test where you specify multiple entries and ensure those are used appropriately.

Reply:
Comments added and text expanded to multiple ports and to test the `--p2p-server-address` parameter better as well.